recipes/pyjama/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
Since the GitHub tarball includes this file, you shouldn't include it manually in the recipe.

Reply:
Still need you to remove this file since the tarball contains the file (and presumably that one will stay the correct one)

Reply:
Done! Thanks